client/web/src/components/externalServices/externalServices.tsx

```diff
@@ -1124,6 +1185,7 @@ export const codeHostExternalServices: Record<string, AddExternalServiceOptions>
     srcservegit: SRC_SERVE_GIT,
     gitolite: GITOLITE,
     git: GENERIC_GIT,
+    ...(window.context.experimentalFeatures.perforce === 'enabled' ? { perforce: PERFORCE } : {}),
```

Comment:
I think this can be empty

```suggestion
    ...(window.context.experimentalFeatures?.perforce === 'enabled' ? { perforce: PERFORCE } : {}),
```

Reply:
Nice catch :D